### ISSUE TITLE
Consider all mouse+keyboard interactions

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/a11y-research#69]
We have a very rough fix for preventing mouse+keyboard multi-input bugs, initial implementation is in 
https://github.com/phetsims/scenery/issues/713.

But there are still inconsistencies in the way this is handled, particularly across platforms. There is some discussion in https://github.com/phetsims/scenery/issues/713 about this, but https://github.com/phetsims/friction/issues/50 is another issue related to this.

This issue is to thoroughly discuss how interacting with a sim with mouse + keyboard should behave.

**Comment by jessegreenberg** [comment]
Making a list of issues that this will touch:
- https://github.com/phetsims/friction/issues/50

NOTE: This comment will grow.

**Comment by terracoda** [comment]
@jessegreenberg, wondering if Ohm's Law issue https://github.com/phetsims/ohms-law/issues/72, especially regarding @samreid 's comment

> When scenery starts processing inputs on the 2nd slider, can we just set the focus to the second slider programmatically? (https://github.com/phetsims/ohms-law/issues/72#issuecomment-322067878)

**Comment by jessegreenberg** [comment]
Thanks @terracoda that is also related to this issue, we should also consider how description should behave when going back and forth between mouse and keyboard/AT interaction.

**Comment by zepumph** [comment]
Marking for a11y meeting to keep track of it.
